.scalafmt.conf

```diff
@@ -1 +1,4 @@
-version = 2.7.5
\ No newline at end of file
+version = 3.7.3
```

Comment:
Newest stable version seems to be [3.8.3](https://github.com/scalameta/scalafmt/releases), have you tried that?

Reply:
Nevermind, does _not_ seem to work

Reply:
No I haven't but so we are at least synchronous with simona...